### ISSUE TITLE
Profile update

### ISSUE THREAD

**Issue by sivagollapalli** [joshsoftware/code-curiosity#78]
Currently, there is no way to update my profile information. It would be better if we add it because on Github user can change primary email ID. Thoughts, please?

**Comment by gautamrege** [comment]
Good idea. Lower on our priority of fixes but would be great if you can send a PR
